src/main/i18n/index.js

```diff
@@ -0,0 +1,4 @@
+import i18n, { DEFAULT_LOCALE } from '../../locales'
```

Comment:
My langugage JSON is in Vue-i18n's format, which manages translations by tree. This makes translations much more clear.

Currently no other plugin supports this format. If you replace Vue-i18n to other plugin, I may need to maintain more than one translation files, or totally rewrite them.

Even though Vue mainly runs on renderer process, it can also run in main process. I use Vue for serving Vue-i18n only. I've tested on my environment and it works well.